docs/en/stock-management/movement.exit.md

```diff
@@ -2,29 +2,221 @@
+      items.  These are typical items such as laxatives, alcohol,
+      anti-bacterial creams, etc.  In short, these are stock items for daily
+      use that are difficult to quantify in advance to be part of a
+      prescription issued for a specific patient.  This category may also
+      include resuscitation and emergency use products that cannot wait for
+      the patient to pay first at the counter before being served at the
+      pharmacy.
 
-Note that stock exits to a patient, service, or recording a loss effectively remove stock items from the enterprise.  The only way to replenish stock after an exit is made is to perform a stock adjustment.
+    - **Transfer to other Depots or dispensing pharmacies**: Normally, patients
+      are served directly at a dispensing pharmacy (or dispensary) when the
+      dispensing pharmacy receives prescription (that have been paid for) at
+      the counter, by means of a receipt ("proof of payment"), presented by
+      the patient or a nurse.  Stock Pharmacies are usually the source of
+      stock that is transferred to Dispensing Pharmacies.
 
-# Average Monthly Consumption (AMC)
+    - **Losses**: Losses are also taken into account here.  This the losses
+      themselves, breakages, as well as lapses???.  It is advisable to remove
+      the lost items from the stock inventory.
 
-The Average Monthly Consumption (AMC) in BHIMA is _currently_ defined as follows:
+ - The movements of stock out of the dispensing pharmacy occur in the
+   following cases:
 
-```sh
-AMC = SUM(quantity consumed) / COUNT(days with a stock consumption event)
-```
+    - **Stock items transferred to patients** [^1].  As above, patients with
+      prescriptions with proof of payment are directly served at the
+      dispensing pharmacy. These outflows are recorded, valued and constitute,
+      to a large extent, the largest proportion of the source of the drugs
+      (stock) consumed / sold.  These stock movements should be carried in the
+      expense part of the operating account.
 
-Technically, this is not a complete metric.  This is being tracked in [#4346](https://github.com/IMA-WorldHealth/bhima/issues/4346).  We are working towards implementing the following:
+    - **Losses**: Like the stock pharmacy, the losses themselves, breakages as
+      well as lapses are the main reasons for causing exit of stock from the
+      usual pharmacy (in terms of loss).
 
-```sh
-AMC = SUM(quantity consumed) / COUNT(days with stock on hand)
-```
+NOTE: When the losses of stock cause the inventory of a stock item to be
+depleted, we call that "Out of Stock" or a "Stock Out".
 
-However, for highly used medications, both calculations should approach the same value.
+Stock exits to a patient, service, or to loss effectively removes stock items
+from the enterprise.  The only way to replenish stock after an exit is made is
+to receive more stock by an Entry.
```

Comment:
This is only somewhat true, but maybe we don't need to mention it here.  You can also do a stock adjustment to increase the stock of a lot that you have used up completely.

I'm not sure if that should be mentioned here though.